supply/lib/supply/setup.rb

```diff
@@ -40,34 +39,58 @@ def store_metadata(listing)
       end
     end
 
-    def download_images(listing)
+    def download_images(listing, image_format = "png", remove_alpha = "false")
```

Comment:
Any reasons why `remove_alpha` is a string and not a boolean here?

Reply:
Good catch! My bad, was using `remove_alpha` as a `boolean` (https://github.com/fastlane/fastlane/pull/14322/files#diff-674b785c2e89b32d433060b131c0cf28R72), but passing it as a string. Fixed it here - https://github.com/fastlane/fastlane/pull/14322/commits/fd93d52e0d23e70b304525b6bd27e0c87ed6a6cd